src/components/page/register/RegisterForm.tsx

```diff
@@ -0,0 +1,178 @@
+                    "userid": userid,
+                    "password": password,
+                    "firstName": firstName,
+                    "lastName": lastName,
+                    "email": email,
+                    //"address": address,
+                    //"birthday": birthday,
+                    "admin": false,
+                })
+            })
+                .then(async (res) => {
+                    if (!res.ok) {
+                        throw await res.json();
+                    }
+
+                    return res.json().then((result) => {
+                        console.log(result);
+
+                    })
+                })
+                .catch((error) => {
+                    console.log("error occured ", error);
+                })
+
+        } else {
+            console.log("Passwords do not match!");
+        }
+    }
+
+    /** NOTICE:
+     *  Ignore error messages on the TextFields
+     */
+
+    return (
+        <Box    // Background
+            sx={{
+                ...SX_MASKS[1]("bottom"),
+                position: "relative",
+                overflow: "hidden",
+                backgroundColor: `${COLOR_PALLETE[0]}C0`,
+                backgroundSize: "cover",
+                backgroundRepeat: "no-repeat",
+                backgroundPositionX: "right",
+                backgroundPositionY: "90%",
+                display: "flex",
+                justifyContent: "center",
+                alignItems: "center",
+                padding: "25%",
+                height: "120vh",
+                zIndex: 0,
+            }}
+        >
+            <div    // Form Contents
+                style={{
+                    flexGrow: 1,
+                    backgroundColor: `white`,
+                    // paddingBottom: "3vw",
+                    borderRadius: "2rem",
+                    borderWidth: "5px",
+                    borderColor: COLOR_PALLETE[1],
+                    borderStyle: "solid",
+                }}
+            >
+                <Container
+                    sx={{
+                        textAlign: "center",
+                        pb: 10,
+                    }}
+                    maxWidth="sm"
+                >
+                    <Typography
+                        sx={{ mt: 3, fontWeight: "700" }}
+                        variant="h2"
+                        color={COLOR_PALLETE[1]}
+                    >
+                        Registeration
+                    </Typography>
+
+                    <form className="registerForm" onSubmit={handleSubmit}>
+                        <Grid container spacing={5} sx={{
+                            marginTop: "0%",
+                        }}>
+                            <Grid item xs={6}>
+                                <TextField
+                                    required
+                                    label="First Name"
+                                    onInput={e => setFirstName(e.target.value)}
```

Comment:
You can use `onChange` instead of `onInput` so that e would have `e.target.value`